runtime/common/src/apis.rs

```diff
@@ -340,22 +340,24 @@ macro_rules! impl_runtime_apis_plus_common {
 							let without_base_extrinsic_weight = true;
 
 
-							// Estimated encoded transaction size must be based on the heaviest transaction
-							// type (EIP1559Transaction) to be compatible with all transaction types.
+							// Estimated encoded transaction size must be based on the transaction
+							// type (TransactionData) to be compatible with all transaction types.
+							// TODO: remove, since we will get rid of base_cost
 							let mut estimated_transaction_len = data.len() +
-							// pallet ethereum index: 1
-							// transact call index: 1
-							// Transaction enum variant: 1
-							// chain_id 8 bytes
-							// nonce: 32
-							// max_priority_fee_per_gas: 32
-							// max_fee_per_gas: 32
-							// gas_limit: 32
-							// action: 21 (enum varianrt + call address)
-							// value: 32
-							// access_list: 1 (empty vec size)
-							// 65 bytes signature
-							258;
+								// pallet ethereum index: 1
+								// transact call index: 1
+								// Transaction enum variant: 1
+								// chain_id 9 bytes (some varient)
+								// nonce: 32
+								// max_priority_fee_per_gas: 33 (some varient)
+								// max_fee_per_gas: 33 (some varient)
+								// gas_price: 1 (none varient)
+								// gas_limit: 32
+								// action: 21 (enum varianrt + call address)
+								// value: 32
+								// access_list: 1 (empty vec size)
+								// 65 bytes signature
+								261;
```

Comment:
This is going to be removed in a follow-up PR, we should revert the changes.

Reply:
Reverted will check if there's issues in the CI